src/App.tsx

```diff
@@ -11,18 +10,15 @@ const Pokemon = lazy(
 );
 function App() {
   return (
-    <div
-      className="relative flex flex-col justify-center items-center w-full h-full object-fill  overflow-clip
-    "
-      object-fill
-    >
-      <div className="absolute z-0 overflow-auto object-cover ">
-        <Background />
-      </div>
-      <Header />
+    <div className="relative flex flex-col justify-center items-center w-screen h-screen object-fill ">
+      <Background />
```

Comment:
Few things here

1. I don't think would require the background component here, this image can just be applied as a background image for the top div.
2. The object fit properties apply to image tags .. it won't apply to other elements
3. As an alternative to object fit you can use background-size and/or background-position to adjust your images when applying images to div backgrounds
4. I don't think justify-center is required here because the child are already filling up the page. the items-center makes sense so that can stay.
5. You can remove the relative class as its not gonna be required because we are removing the background component that had absolute positioning.

```suggestion
    <div className="flex flex-col items-center w-screen h-screen">
```